lottie/src/main/java/com/airbnb/lottie/LottieTask.java

```diff
@@ -23,6 +26,8 @@
  */
 public class LottieTask<T> {
 
+  private static final Executor EXECUTOR = Executors.newCachedThreadPool();
```

Comment:
This seems like something people might want to customize. The easiest way to enable that would be to make it public and non final
that's an approach i started using to set epoxy defaults with the least amount of boilerplate